### ISSUE TITLE
NumberFormatException in EditIndividualWidget.getWidgetCount2

### ISSUE THREAD

**Issue by tianxiaogu** [wistein/TourCount#15]
Can find a couple of such exceptions.
You probably need to implement a safe parseInt helper function. 

```
// CRASH: com.wmstein.tourcount (pid 3327) (elapsed nanos: 14895627327916)
// Short Msg: java.lang.NumberFormatException
// Long Msg: java.lang.NumberFormatException: Invalid int: "18616112460374038731"
// Build Label: google/hammerhead/hammerhead:6.0.1/M4B30Z/3437181:user/release-keys
// Build Changelist: 3437181
// Build Time: 1478203422000
// java.lang.NumberFormatException: Invalid int: "18616112460374038731"
// 	at java.lang.Integer.invalidInt(Integer.java:138)
// 	at java.lang.Integer.parse(Integer.java:413)
// 	at java.lang.Integer.parseInt(Integer.java:367)
// 	at java.lang.Integer.parseInt(Integer.java:334)
// 	at com.wmstein.tourcount.widgets.EditIndividualWidget.getWidgetCount2(EditIndividualWidget.java:204)
// 	at com.wmstein.tourcount.EditIndividualActivity.saveData(EditIndividualActivity.java:272)
// 	at com.wmstein.tourcount.EditIndividualActivity.onOptionsItemSelected(EditIndividualActivity.java:333)
// 	at android.app.Activity.onMenuItemSelected(Activity.java:2914)
// 	at android.support.v4.app.FragmentActivity.onMenuItemSelected(FragmentActivity.java:408)
// 	at android.support.v7.app.AppCompatActivity.onMenuItemSelected(AppCompatActivity.java:195)
// 	at android.support.v7.view.WindowCallbackWrapper.onMenuItemSelected(WindowCallbackWrapper.java:113)
// 	at android.support.v7.app.AppCompatDelegateImplV9.onMenuItemSelected(AppCompatDelegateImplV9.java:679)
// 	at android.support.v7.view.menu.MenuBuilder.dispatchMenuItemSelected(MenuBuilder.java:822)
// 	at android.support.v7.view.menu.MenuItemImpl.invoke(MenuItemImpl.java:156)
// 	at android.support.v7.view.menu.MenuBuilder.performItemAction(MenuBuilder.java:969)
// 	at android.support.v7.view.menu.MenuBuilder.performItemAction(MenuBuilder.java:959)
// 	at android.support.v7.widget.ActionMenuView.invokeItem(ActionMenuView.java:623)
// 	at android.support.v7.view.menu.ActionMenuItemView.onClick(ActionMenuItemView.java:154)
// 	at android.view.View.performClick(View.java:5204)
// 	at android.view.View$PerformClick.run(View.java:21153)
// 	at android.os.Handler.handleCallback(Handler.java:739)
// 	at android.os.Handler.dispatchMessage(Handler.java:95)
// 	at android.os.Looper.loop(Looper.java:148)
// 	at android.app.ActivityThread.main(ActivityThread.java:5417)
// 	at java.lang.reflect.Method.invoke(Native Method)
// 	at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:726)
// 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:616)
// 

```

**Comment by wistein** [comment]
@tianxiaogu: Thanks a lot for testing.

Fixed in Version 2.3.1

**Comment by tianxiaogu** [comment]
@wistein Thanks a lot for taking time to fix this issue.
